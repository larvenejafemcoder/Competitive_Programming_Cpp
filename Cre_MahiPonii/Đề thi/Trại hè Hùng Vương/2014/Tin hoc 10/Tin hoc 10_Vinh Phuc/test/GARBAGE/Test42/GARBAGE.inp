NSWUSEUNDSEDEUWSWWWDDESNENUNWUDUNDEWDSENSDNWNNSSSUSDWUNWUWNWDNUESSWDESSENWN
UUDEUNUNSWUUUWNWUWESDSNUEUUNWDUSSEUUWEDNNUNUDDNUDNDSSSUDW
EWUDUDUWWEUUWSNEWWNSNEDUDENNWNSNSUWUSWDUSWDNDESNUSDDDDDDENEDNWUNDWWEDSDESWWDDWESSWWWEDNSDEDUEUN
EWSDSENUESUWWDUWWDUDSSNDNNNNWSNUWUUNNEEDWEWUWUUSWEWUUW
NSEEWN
SDWDNUDDSWUUDE
D 89
