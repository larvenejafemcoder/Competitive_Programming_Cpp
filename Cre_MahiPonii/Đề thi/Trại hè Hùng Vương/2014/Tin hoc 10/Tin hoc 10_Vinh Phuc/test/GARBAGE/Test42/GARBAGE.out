37949591876005217911839171207527959389621960187676822908272912785370303487248524474729711093137933823893636001125741468103752629017635022547819894761
